CMIP6/Guide/dataUsers.md

```diff
@@ -130,12 +130,12 @@ Information is under preparation describing the governance of the following:
 [activityIdJson]: https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_activity_id.json
 [experimentIdJson]: https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_experiment_id.json
 [experimentIdhtml]: https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html
-[cmip6DataRequestCog]: https://www.earthsystemcog.org/projects/wip/CMIP6DataRequest
+[cmip6DataRequest]: https://cmip6dr.github.io/Data_Request_Home
 [cfConventionsPage]: http://cfconventions.org/
 [cmip6GlobalAttGoogleDoc]: http://goo.gl/v1drZl
 [cmip6GridGoogleDoc]: http://goo.gl/1oA7bO
-[cmip6PressureLevelsPdf]: https://www.earthsystemcog.org/site_media/projects/wip/CMIP6_pressure_levels.pdf
-[cmip6TimeAveragesCog]: https://www.earthsystemcog.org/projects/wip/time_averages
+[cmip6PressureLevelsPdf]: https://cmip6dr.github.io/Data_Request_Home/Documents/CMIP6_pressure_levels.pdf
+[cmip6TimeAveragesCog]: https://web.archive.org/web/20201020095856/https://www.earthsystemcog.org/projects/wip/time_averages
```

Comment:
Ported to WIP pages: https://wcrp-cmip.github.io/WGCM_Infrastructure_Panel/CMIP6/time_and_area_averaging.html